integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/standalone/soap/SOAPApisTestCase.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.choreo.connect.tests.testcases.standalone.soap;
+
+import org.apache.http.HttpStatus;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.choreo.connect.tests.common.model.API;
+import org.wso2.choreo.connect.tests.common.model.ApplicationDTO;
+import org.wso2.choreo.connect.tests.context.CCTestException;
+import org.wso2.choreo.connect.tests.util.ApictlUtils;
+import org.wso2.choreo.connect.tests.util.HttpResponse;
+import org.wso2.choreo.connect.tests.util.HttpsClientRequest;
+import org.wso2.choreo.connect.tests.util.TestConstant;
+import org.wso2.choreo.connect.tests.util.TokenUtil;
+import org.wso2.choreo.connect.tests.util.Utils;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class SOAPApisTestCase {
+
+    private final String context = "/soap/1.0.0";
+    private final String version = "1.0.0";
+    private static final String SOAP_ACTION = "http://mockbackend:2340/phoneverify/query/CheckPhoneNumber";
+
+    private String testKey;
+
+    @BeforeClass
+    public void deployApi() throws Exception {
+        ApictlUtils.login("test");
+        ApictlUtils.deploySampleProject("SampleSOAPApi", "test");
```

Comment:
Shall we move this here? https://github.com/wso2/product-microgateway/blob/e4b94fb5105e109218ffdd467c6b99ba471e950d/integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/setup/standalone/CcWithBackendTlsAndCorsDisabledWithOPA.java#L65 Will reduce the waiting time.

Reply:
fixed in https://github.com/wso2/product-microgateway/pull/2972/commits/5c91048f9fa451e4b2f2d91dac704d3f6b755cd3